backend/copr_backend/helpers.py

```diff
@@ -452,21 +450,6 @@ def uses_devel_repo(front_url, username, projectname, project=None):
     return bool(project.get("devel_mode", False))
 
 
-# def log(lf, msg, quiet=None):
-#     if lf:
-#         now = datetime.datetime.utcnow().isoformat()
-#         try:
-#             with open(lf, "a") as lfh:
-#                 fcntl.flock(lfh, fcntl.LOCK_EX)
-#                 lfh.write(str(now) + ":" + msg + "\n")
-#                 fcntl.flock(lfh, fcntl.LOCK_UN)
-#         except (IOError, OSError) as e:
-#             sys.stderr.write(
-#                 "Could not write to logfile {0} - {1}\n".format(lf, str(e)))
-#     if not quiet:
-#         print(msg)
-#
-
```

Comment:
An unrelated change but it only removes a commented code, so it's probably no problem.

Reply:
Yes. The commented code uses utcnow(). I was torn between updating the commented code and deleting it. I choose the later as we did not use this code for ages.